sharded_queue/drivers/fifottl.lua

```diff
@@ -259,6 +259,9 @@ function method.put(args)
         }
     end)
 
+    log.info(("[%d] put %d"):format(
```

Comment:
Add `args.options.log_requests = true|false` and a guard for optional logging (at each call).